### PR TITLE
feat: Add HowTo change customer billing

### PR DESCRIPTION
**Pull request by andersc2022**
Adding guide for change company information such as adress, contactperson, PO number.

### CODE REVIEW

File: docs/howto/getting-started/change-cust-billing.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# How to change company information in your account
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:1]: "Changing account billing data".

File: docs/howto/getting-started/change-cust-billing.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Via {{gui}} <https://{{gui_domain}}> you can change contact person, adress, company name, purchase order or add c/o adress. This guide will show you how.
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:3]:
```markdown
Via the [{{gui}}](https://{{gui_domain}}), you ...
```

Also, typo: missing "d" in "address" (twice).

Also, remove the sentence "This guide will show you how"; it is obvious and redundant.

File: docs/howto/getting-started/change-cust-billing.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Prerequisites
+
+You need a Cleura account.
+
+[Guide how to create a new account](https://docs.cleura.cloud/howto/getting-started/create-account)
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:9]:
```markdown
You need a {{brand}} account, as explained in [this guide](../create-account/)].
```

File: docs/howto/getting-started/change-cust-billing.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## How to change in Cleura Controlpanel
+
+To get started, navigate to <https://{{gui_domain}}>.
+Log in and click on the _Profile_ button at the top right like picture below.
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:14]: Remove "like picture below"; it is grammatically incorrect and also redundant.

File: docs/howto/getting-started/change-cust-billing.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+![picture-profile](assets/change-cust-billing/picture-profile.png)
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:16]: "picture-profile" is not useful as [alt text](https://accessibility.huit.harvard.edu/describe-content-images); replace this with a proper image description.

File: docs/howto/getting-started/change-cust-billing.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+You will now see Account settings page.
+Choose _Customer Info_ tab in which you can change and manage your Customer information.
+
+![picture-customerinfo](assets/change-cust-billing/picture-customerinfo2.png)
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:21]: Again, use sensible alt text.

File: docs/howto/getting-started/change-cust-billing.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+Finished your changes by clicking the green _Update_ button in the bottom
+
+![picture-changecustomerinfo](assets/change-cust-billing/picture-changecustomerinfo2.png)
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:25]: Again, use sensible alt text.

File: docs/howto/getting-started/change-cust-billing.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## How to change in Cleura Controlpanel
+
+To get started, navigate to <https://{{gui_domain}}>.
+Log in and click on the _Profile_ button at the top right like picture below.
+
+![picture-profile](assets/change-cust-billing/picture-profile.png)
+
+You will now see Account settings page.
+Choose _Customer Info_ tab in which you can change and manage your Customer information.
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:19]:
```markdown
You will now see the _Account settings_ page. Choose the _Customer Info_ tab, 
in which you can change and manage your customer information.
```

File: docs/howto/getting-started/change-cust-billing.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+![picture-customerinfo](assets/change-cust-billing/picture-customerinfo2.png)
+
+Finished your changes by clicking the green _Update_ button in the bottom
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:23]: "Finish". Also, add a full-stop at the end of the sentence.

File: docs/howto/getting-started/change-cust-billing.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+![picture-changecustomerinfo](assets/change-cust-billing/picture-changecustomerinfo2.png)
+
+## **How to change organization number**
+
+This can not be done in {{gui}}
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:29]:
```markdown
You can not change your organization number (for business accounts) 
or personal number (for individual accounts) in {{gui}}.
```

File: docs/howto/getting-started/change-cust-billing.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+You will now see Account settings page.
+Choose _Customer Info_ tab in which you can change and manage your Customer information.
+
+![picture-customerinfo](assets/change-cust-billing/picture-customerinfo2.png)
+
+Finished your changes by clicking the green _Update_ button in the bottom
+
+![picture-changecustomerinfo](assets/change-cust-billing/picture-changecustomerinfo2.png)
+
+## **How to change organization number**
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:27]: This is already a heading (`##`); it doesn't need additional boldface (`**`).

Also, make it "Changing an account's organization number".

File: docs/howto/getting-started/change-cust-billing.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+This can not be done in {{gui}}
+
+This requires that you send in a [transfer form](https://kb.citynetwork.eu/kb/how-to-articles/general/transfer-account-ownership-to-another-person-or-organisation) to support@citycloud.com.
```
Comment (fghaas) [inline docs/howto/getting-started/change-cust-billing.md:31]: We don't backlink from the new docs to the old knowledge base. Add the transfer form as an asset like you did with the screenshots, and link to that.

Also, use:

```markdown
to [support@{{brand_domain}}](mailto:support@{{brand_domain}}).
```

**Review by fghaas — CHANGES_REQUESTED** [review]
Thanks; a few issues with this one. Please fix, amend your commit, and then force-push to your topic branch.

**Comment by fghaas** [comment]
Also, please address the issue with the commit message (the line in its body is too long; it just needs a linewrap).

**Comment by fghaas** [comment]
A few changes are still needed here:

* This doesn't belong in the "Getting started" category; instead there should be a separate how-to category for account management and billing.
* One of the screen shots is redundant and can be removed.
* The PR needs a rebase on current `main`.

I've applied all these changes in 711e7d61651b0137b27b8ac27e1440ea7bbc4cd4 now and will ask @colder-is-better for a final review before we can merge this.

File: docs/howto/account-billing/change-billing-data.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
+# Changing your account billing data
+
+Via [{{gui}}](https://{{gui_domain}}) you can change the contact
+person, address, company name, and purchase order number associated
+with your account.
+
+To get started, navigate to <https://{{gui_domain}}>.  Log in and
```
Comment (colder-is-better) [inline docs/howto/account-billing/change-billing-data.md:7]: Just an extra space in "[...]<https://{{gui_domain}}>.  Log in and[...]" :)

File: docs/howto/account-billing/change-billing-data.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+
+![The "Customer Info" tab with a green "Update" button to complete the
+changes](assets/picture-changecustomerinfo2.png)
+
+Finish your changes by clicking the green _Update_ button at the
+bottom.
+
+## Changing an account's organization number
+
+You can not change your organization number (for business accounts) or
```
Comment (colder-is-better) [inline docs/howto/account-billing/change-billing-data.md:25]: As in most cases, "can not" should be written as "cannot".

**Review by colder-is-better — COMMENTED** [review]
Looks good --- I have only two pico-nits.

File: docs/howto/account-billing/change-billing-data.md (hunk 13 above)
Comment (fghaas) [reply docs/howto/account-billing/change-billing-data.md]: Fixed, thanks.

File: docs/howto/account-billing/change-billing-data.md (hunk 14 above)
Comment (fghaas) [reply docs/howto/account-billing/change-billing-data.md]: Done.

**Review by fghaas — APPROVED** [review]
Never mind @colder-is-better; I'll just go ahead and approve & merge this one now.

**Pull request merged**